app/service.js

```diff
@@ -0,0 +1,184 @@
+         return socialLinks;
+     }
+     
+     tools = [{
+            url: 'https://developer.mozilla.org/en-US/docs/Web/HTML',
+			title: 'HTML',
+			img: imgRoot + 'html.png',
+            type: 'tool'
+            },{
+            url: 'https://developer.mozilla.org/en-US/docs/Web/CSS',
+			title: 'CSS',
+			img: imgRoot + 'css.png',
+            type: 'tool'
+            },{
+            url: 'http://getbootstrap.com/',
+			title: 'Bootstrap',
+			img: imgRoot + 'bootstrap.png',
+            type: 'tool'
+            },{
+            url: 'http://jquery.com/',
+			title: 'jQuery',
+			img: imgRoot + 'jquery.png',
+            type: 'tool'                                  
+            },{
+            url: 'https://angularjs.org/',
+			title: 'AngularJS',
+			img: imgRoot + 'angular.png',
+            type: 'tool'
+            },{
+            url: 'https://www.firebase.com/',
+			title: 'Firebase',
+			img: imgRoot + 'firebase.png',
+            type: 'tool'                                      
+            },{
+            url: 'https://developer.mozilla.org/en/docs/Web/JavaScript',
+			title: 'JavaScript',
+			img: imgRoot + 'javascript.png' ,
+            type: 'tool'                                     
+            },{
+            url: 'https://msdn.microsoft.com/en-us/library/kx37x362.aspx',
+			title: 'C#',
+			img: imgRoot + 'csharp.png',
+            type: 'tool'                                  
+            },{
+            url: 'https://github.com/',
+			title: 'GitHub',
+			img: imgRoot + 'github.png',
+            type: 'tool'                                     
+            },{
+            url: 'https://www.microsoft.com/net/default.aspx',
+			title: 'Microsoft.NET',
+			img: imgRoot + 'dotnet.png',
+            type: 'tool'                                       
+            },{
+            url: 'https://www.visualstudio.com/',
+			title: 'Visual Studio',
+			img: imgRoot + 'vs.png',
+            type: 'tool'                                      
+            },{
+            url: 'https://www.jetbrains.com/idea/',
+			title: 'Intellij IDEA',
+			img: imgRoot + 'idea.png',
+            type: 'tool'                                                                 
+            },{
+            url: "https://www.java.com/",
+			title: "Java",
+			img: imgRoot + 'java.png',
+            type: 'exp'
+            },{
+            url: "http://developer.android.com/index.html",
+			title: "Android Studio",
+			img: imgRoot + 'as.png',
+            type: 'exp'
+            },{
+            url: "https://nodejs.org/",
+			title: "Node.JS",
+			img: imgRoot + 'nodejs.png',
+            type: 'exp'
+          }];
+     
+   
+     this.getTools = function(value){
+         var list = [];
+         for (var i = 0; i < tools.length; i++) {
+             if (tools[i].type == value) {
+                 list.push(tools[i]);
+             };
+         };
+         return list;
+     };
+     
+     
+     skills = [{
```

Comment:
This will create global variable and pollutes global namespace - considered a bad practice. In this case you need a local variable and use the `var` keyword.